deploy/olm-catalog/tektoncd-pipeline-operator/0.3.1/tektoncd-pipeline-operator.v0.3.1.clusterserviceversion.yaml

```diff
@@ -3,21 +3,42 @@ kind: ClusterServiceVersion
 metadata:
   annotations:
     capabilities: Basic Install
+    description: An operator to install TektonCD Pipeline
+    containerImage: quay.io/openshift-pipeline/tektoncd-pipeline-operator:v0.3.1-3
+    createdAt: 2019-03-15T19:44:21Z
+    certified: "false"
+    support: Red Hat, Inc.
+    repository: https://github.com/openshift/tektoncd-pipeline-operator
     alm-examples: |
       [
         {
           "apiVersion": "tekton.dev/v1alpha1",
           "kind": "Install",
           "metadata": {
-            "name": "example-install",
-            "namespace": "tekton-pipelines"
-        },
+            "name": "example-install"
```

Comment:
why was the namespace removed?

Reply:
based on this comment in community operators pr
https://github.com/operator-framework/community-operators/pull/296#discussion_r277818443

in OKD the namespace into which it is installed will be shown in webconsole yaml editor